RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+Reducer.swift

```diff
@@ -33,8 +64,12 @@ public struct DisplayEntitiesControlledByMnemonic: Sendable, FeatureReducer {
 		switch viewAction {
 		case .appeared:
 			.none
-		case .tapped:
-			.send(.delegate(.openDetails))
+		case .navigateButtonTapped:
+			if state.mnemonicIsMissingNeedsImport {
+				.send(.delegate(.importMissingMnemonic))
+			} else {
+				.send(.delegate(.displayMnemonic))
+			}
```

Comment:
I would probably prefer to have two different view actions, unless there are strong reasons to delay the check to the reducer

Reply:
I had that but I thought the `action` closure in view got ugly, but sure can switch to that